src/opcode.rs

```diff
@@ -152,20 +152,13 @@ opcode!(
         sqe.ioprio = ioprio;
         sqe.__bindgen_anon_2.addr = iovec as _;
         sqe.len = len;
-        sqe.__bindgen_anon_1.off = offset64 as _;
+        sqe.__bindgen_anon_1.off = offset;
         sqe.__bindgen_anon_3.rw_flags = rw_flags;
         sqe.__bindgen_anon_4.buf_group = buf_group;
         Entry(sqe)
     }
 );
 
-impl Readv {
-    #[inline]
-    pub const fn offset(self, offset: libc::off_t) -> Self {
-        self.offset64(offset as libc::off64_t)
-    }
-}
-
```

Comment:
ummm, what's the reason to remove the setter of Readv.offset? How can a user pass an offset to preadv(2) then?

Reply:
Ahhh, never mind, I found the builder pattern implemented by the opcode! macro